src/js/containers/Institutions.jsx

```diff
@@ -24,15 +22,9 @@ export class InstitutionContainer extends Component {
 export function mapStateToProps(state) {
   const { institutions } = state.app.institutions
 
-  const {
-    filings,
-    submission,
-    error,
-    filingPeriod
-  } = state.app
+  const { filings, error, filingPeriod } = state.app
```

Comment:
@wpears - this bit is new. While doing some testing it seemed like `submission` never existed at this point. Plus we [can do this](https://github.com/cfpb/hmda-platform-ui/pull/745/files#diff-86bed846ff53cc9ec2c51bcb085621d4R48) anyway. Let me know if I'm missing something here.

Reply:
The submission exists once it's been created and once the submission page is visited, but should always result in the same value as this method.

That said, this method requires that submissions are returned in reverse sorted order (which is currently true, but is undocumented and subject to change).

I'd be more comfortable with a function that takes the submissions array, loops over it, it returns the one with the highest `sequenceNumber`

Reply:
I just brought it back, for now, was easier to do (78511bd).